src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(crateReport(readFromFile(fromFileName)));
+
+        } catch (IOException e) {
+            throw new RuntimeException(e + " Can not write to file " + toFileName);
+        }
```

Comment:
Let's extract it to the separate `private` method. Remember from `checklist`:

**Divide program logic.** For example: “write to file” or “create report” - are a separate operations. So we should create separate methods for them.

Also, let's build a single result String, not the array.

```suggestion
           writeToFile(createReport(readFromFile(fromFileName)));
```